Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-event/2.12.0-SNAPSHOT/jspwiki-event-2.12.0-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-event/2.11.0/jspwiki-event-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.event.WikiEngineEvent  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.event.WikiEvent (<- org.apache.wiki.event.WikiEvent)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int INITIALIZED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int STOPPED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int INITIALIZING
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int SHUTDOWN
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiEngineEvent(java.lang.Object, int)
	===  UNCHANGED METHOD: PUBLIC FINAL java.lang.String eventName()
	===  UNCHANGED METHOD: PUBLIC FINAL java.lang.String getTypeDescription()
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isValidType(int)
	===  UNCHANGED METHOD: PROTECTED void setType(int)
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.apache.wiki.event.WikiEvent  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.util.EventObject (<- java.util.EventObject)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int UNDEFINED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int ERROR
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiEvent(java.lang.Object, int)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiEvent(java.lang.Object, int, java.lang.Object[])
	===  UNCHANGED METHOD: PUBLIC java.lang.String eventName()
	===  UNCHANGED METHOD: PUBLIC java.lang.Object getArg(int, java.lang.Class<T><T>)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC java.lang.Object[] getArgs()
	===  UNCHANGED METHOD: PUBLIC java.lang.Object getSrc()
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC int getType()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getTypeDescription()
	===  UNCHANGED METHOD: PUBLIC long getWhen()
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isValidType(int)
	===  UNCHANGED METHOD: PROTECTED void setType(int)
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED ENUM: PUBLIC FINAL org.apache.wiki.event.WikiEventEmitter  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.lang.Enum (<- java.lang.Enum)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.event.WikiEventEmitter INSTANCE
	===  UNCHANGED METHOD: PUBLIC STATIC void attach(org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.event.WorkflowEvent fireWorkflowEvent(java.lang.Object, int)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.event.WorkflowEvent fireWorkflowEvent(java.lang.Object, int, java.lang.Object[])
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.event.WikiEventEmitter get()
	===  UNCHANGED METHOD: PUBLIC STATIC void register(org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.event.WikiEventEmitter valueOf(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.event.WikiEventEmitter[] values()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.event.WikiEventListener  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.EventListener
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void actionPerformed(org.apache.wiki.event.WikiEvent)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.event.WikiEventManager  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean addWikiEventListener(java.lang.Object, org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC STATIC void fireEvent(java.lang.Object, org.apache.wiki.event.WikiEvent)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.event.WikiEventManager getInstance()
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Set<org.apache.wiki.event.WikiEventListener><org.apache.wiki.event.WikiEventListener> getWikiEventListeners(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.UnsupportedOperationException
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isListening(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean removeWikiEventListener(java.lang.Object, org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean removeWikiEventListener(org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC STATIC void shutdown()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.event.WikiPageEvent  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.event.WikiEvent (<- org.apache.wiki.event.WikiEvent)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_UNLOCK
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int POST_SAVE_END
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int POST_SAVE_BEGIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int POST_SAVE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_DELETED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRE_TRANSLATE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRE_SAVE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRE_TRANSLATE_BEGIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRE_TRANSLATE_END
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRE_SAVE_BEGIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_REQUESTED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRE_SAVE_END
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_DELIVERED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int POST_TRANSLATE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_REINDEX
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int POST_TRANSLATE_END
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_DELETE_REQUEST
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int POST_TRANSLATE_BEGIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_LOCK
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiPageEvent(java.lang.Object, int, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String eventName()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getPageName()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getTypeDescription()
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isValidType(int)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.event.WikiPageRenameEvent  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.event.WikiPageEvent (<- org.apache.wiki.event.WikiPageEvent)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PAGE_RENAMED
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiPageRenameEvent(java.lang.Object, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String eventName()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getNewPageName()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getOldPageName()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getTypeDescription()
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isValidType(int)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.event.WikiSecurityEvent  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.event.WikiEvent (<- org.apache.wiki.event.WikiEvent)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGOUT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_CREDENTIAL_EXPIRED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int GROUP_ADD
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PROFILE_SAVE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int ACCESS_ALLOWED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int GROUP_REMOVE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_ACCOUNT_EXPIRED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int ACCESS_DENIED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_ASSERTED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_FAILED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_INITIATED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PRINCIPAL_ADD
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int SESSION_EXPIRED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PROFILE_NAME_CHANGED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_ANONYMOUS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LOGIN_AUTHENTICATED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int GROUP_CLEAR_GROUPS
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiSecurityEvent(java.lang.Object, int, java.security.Principal, java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiSecurityEvent(java.lang.Object, int, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC java.lang.String eventName(int)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object getPrincipal()
	===  UNCHANGED METHOD: PUBLIC java.lang.Object getTarget()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getTypeDescription()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.event.WorkflowEvent  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.event.WikiEvent (<- org.apache.wiki.event.WikiEvent)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int CREATED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int COMPLETED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DQ_REMOVAL
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int RUNNING
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DQ_ADDITION
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int STARTED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DQ_REASSIGN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int WAITING
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DQ_DECIDE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int ABORTED
	===  UNCHANGED CONSTRUCTOR: PUBLIC WorkflowEvent(java.lang.Object, int)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WorkflowEvent(java.lang.Object, int, java.lang.Object[])
	===  UNCHANGED METHOD: PUBLIC java.lang.String eventName(int)
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
